gradlew

```diff
@@ -130,7 +130,7 @@ fi
 if [ "$cygwin" = "true" -o "$msys" = "true" ] ; then
     APP_HOME=`cygpath --path --mixed "$APP_HOME"`
     CLASSPATH=`cygpath --path --mixed "$CLASSPATH"`
-    
```

Comment:
gradlew and gradlew.bat are automatic created with the cradle plugin.
Please let them unchanged.